eslint.config.mjs

```diff
@@ -4,5 +4,5 @@ export default storyblokLintConfig({
   rules: {
     'no-console': 'off',
```

Comment:
@edodusi I don't think we should have this globally, is a rule that should be disable only on files that require it like playgrounds

Reply:
but if I don't disable this rule I would get an error for every console.log, and we have some in this lib

Reply:
Hi @edodusi the rule should be enforced globally and then disabled in the specific cases that are used on the library by adding:

```
// eslint-disable-next-line no-console
console.log('This console log is inside the lib')
```

You can also add it on the top of the file in case you have several console logs

```
/* eslint-disable no-console */
```

Reply:
I'll do it, be aware that's basically any single relevant file for this lib

Reply:
I take it back: the default apparently is to allow console statements, if I remove this conf I get:
```Unused eslint-disable directive (no problems were reported from 'no-console')```

Reply:
oh because `errors` are tolerated, got it